kafka-connect/build.gradle

```diff
@@ -262,4 +263,4 @@ project(':iceberg-kafka-connect:iceberg-kafka-connect-transforms') {
   test {
     useJUnitPlatform()
   }
-}
```

Comment:
nit: can we revert the unnecessary white space change

Reply:
+1